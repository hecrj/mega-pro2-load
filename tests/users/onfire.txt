9

100
40
32
90
500
260
300
1000
1800

17

1 2 4 8 16 0 0 0 9 17 0 0 0 5 10 0 0 11 0 0 3 6 12 0 0 13 0 0 7 14 0 0 15 0 0

30
4 3 4 6 8 
75
2 1 9
55
3 2 4 9
25
1 1 
40
4 2 4 6 8
100
2 1 9
10
5 4 5 6 8 9
15
1 3
60
0
40
2 6 9
20
1 3 
200
0
25
1 5 
75
1 6
20
2 5 8
30
1 7
30
1 7

-1 1 1

-1 2 2

-1 3 3

-1 4 4

-1 5 5

-1 6 6

-1 7 7

-1 3 8

-2 9

-3 10

-5 2 4

-5 3 8

-1 8 50

-1 9 53

-1 6 61

-1 5 62

-2 63

-3 64

-5 6 100

-5 10 70

-6
