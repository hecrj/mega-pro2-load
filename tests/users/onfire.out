Peticion procesada y servidores
0 1
2 4 
Peticion procesada y servidores
1 1
3 
Peticion procesada y servidores
2 1
1 8 
Peticion procesada y servidores
3 1
1 3 7 
Peticion procesada y servidores
4 17
7 15 
Peticion procesada y servidores
5 3
1 5 10 
Peticion procesada y servidores
6 10
16 
Peticion procesada y servidores
7 2
11 
Peticiones pendientes
7 3 8 10
6 7 7 17
4 5 5 22
Servidores ocupados
7 4 12
15 4 12
16 6 7
Pelicula mas solicitada
2 1
Pelicula mas solicitada
3 2
Peticion procesada y servidores
8 15
1 5 
Peticion procesada y servidores
9 12
3 6 
Peticion procesada y servidores
10 4
7 14 
Peticion procesada y servidores
11 20
13 
Peticiones pendientes
8 8 50 65
9 9 53 65
10 6 61 65
11 5 62 82
Servidores ocupados
1 8 1
3 9 1
5 8 1
6 9 1
7 10 1
13 11 18
14 10 1
Pelicula mas solicitada
6 2
Pelicula mas solicitada
5 1
